force-app/main/default/classes/DisplayMapController.cls

```diff
@@ -0,0 +1,11 @@
+public with sharing class DisplayMapController {
+    @AuraEnabled (cacheable=true)
+    public static List<Public_Event__c> getEventLocations(String eventIdInitial){
+        
+        //for home page
+        //String str = eventNameInitial + '%';
+        //return [SELECT Name, Street__c, City__c, Country__c, Postal_Code__c FROM Public_Event__c WHERE ID =: str];
+
+        return [SELECT Name, Street__c, City__c, Country__c, Postal_Code__c FROM Public_Event__c WHERE ID =: eventIdInitial];
```

Comment:
@t-gul please check if the current user has CRUD permissions
cc: @bmtemakina FYI